fourkitchens/emulsify-gulp
# feat(babel): Add babel to scripts/styleguide-scripts tasks.

# Add babel

## Test
* pull this branch
* `yarn`
* `yarn link`
* open emulsify repo
* `yarn link emulsify-gulp`
* inside of emulsify... `yarn start`
* Verify components' JS in pattern lab works
* Verify inside of `./dist` that components' JS was compiled individually

NOTE: A PR will need to be made against emulsify to update the library paths _and_ (unless we can find another solution) add babel-preset-env as a dev dependency.
